plugins/chat/assets/javascripts/discourse/components/chat-thread.js

```diff
@@ -0,0 +1,21 @@
+import Component from "@glimmer/component";
+import I18n from "I18n";
+import { alias } from "@ember/object/computed";
+import { inject as service } from "@ember/service";
+
+export default class ChatThreadPanel extends Component {
+  @service siteSettings;
+  @service currentUser;
+  @service chat;
+  @service router;
+
+  @alias("chat.activeThread") thread;
```

Comment:
Generally speaking I recommend you just never use alias. I have never encountered a valid use case for it.

Moreover, with glimmer, it's better to just directly use a getter:

```javascript
get thread() {
  return this.chat.activeThread;
}
```

Reply:
Okay no problem :+1: